doc/MediaSigning.xml

```diff
@@ -0,0 +1,689 @@
+            <para>The client should signal AUTHENTIC if all computed hashes are verified
+              successfully against the hashes in the SEI, and the order is preserved and no NALUs
+              are missing.</para>
+            <para>If any missing NALU is detected the client should signal AUTHENTIC WITH MISSING
+              NALUS.</para>
+            <para>If no SEIs are present the client should signal NOT SIGNED.</para>
+            <para>If none of the above applies the video is not authentic and the client should
+              signal NOT AUTHENTIC.</para>
+          </listitem>
+        </itemizedlist>
+      </para>
+    </section>
+    <section>
+      <title>
+        <?oxy_comment_start author="FredrikS" timestamp="20220922T125214+0200" comment="Informative"?>Certificate
+        validation<?oxy_comment_end?>
+      </title>
+      <para>In order to validate the certificate, one follows the certificate chain from the leaf
+        certificate to the root certificate according to the x.509 standard. Since Onvif does not
+        provide a trusted certificate store the client implementor and/or the end user must install
+        trusted root CA certificates in the client. Typically one also need the intermediate CA
+        certificate which is also provided by the device. When verifying old video, it may happen
+        that the certificates have expired as compared to the current date. However, the
+        verification concerns the date that the video was created as such the date should be
+        validated against the claimed recording date rather than the current date. </para>
+    </section>
+    <section>
+      <title>
+        <?oxy_comment_start author="FredrikS" timestamp="20220922T125717+0200" comment="Do we need to add info in Export FF?"?>Exporting
+        a Signed Media<?oxy_comment_end?>
+      </title>
+      <para>In order to be able to verify an exported segment of Signed Media, the client must
+        include the "Send Once" TLVs that are included in the beginning of the mediastream. This can
+        be combined with the Onvif Export File Format in order to provide layered video
+        authentication, the media signing covers authenticity in the media layer where as the Export
+        File Format provides validity on the container level. Even if this Signed Media
+        specification is used, the Onvif Export File Format is still valuable as it provides more
+        information of the provenance of the video as it allows the user to tell the story of the
+        path of the video. </para>
+    </section>
+    <section>
+      <title><?oxy_comment_start author="FredrikS" timestamp="20220922T125248+0200" comment="Informative"?>Dangling
+        End<?oxy_comment_end?>
+      </title>
+      <para>One special consideration is the possbility of dangling ends, i.e. that the SEI frame
+        containing the verification information for a GOP may be very delayed. This can be due to a
```

Comment:
"This can be due to a number of reasons" suggestion "There are a number of reasons for this happening such".  I would insert bullet points to list the reasons. It would read and emprise your points more clearly than one big paragraph.

Reply:
Fixed